### DISCUSSION TITLE
case-studies/the-shadow/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#194] [Website Discussions]
# Der Schatten (The Shadow)

Case Study for a Mystery Thriller Series

https://fcp.cafe/case-studies/the-shadow/

**Comment by latenitefilms** [comment]
This is an amazingly detailed read Florian Duffe! HUGE thanks for sharing all your amazing knowledge with the community!!

**Comment by TomThistlethwaite** [comment]
Great write-up, Florian. Can you elaborate about reels and what you are doing to keep the timeline snappy? My recent 44min series behaved OK but doing 60mins in the past has been problematic. I'm always wondering whether it's the number of items in the project or the duration of the timeline that creates the slowing down in FCP. The volume of material in Der Schatten suggests it is the latter.

**Reply by florian-duffe** [reply]
thank you, sure.

reels:
imagine you segment your timeline into regions of around 20-25 minutes and put them into Compound Clips. Inside the Compounds you drive the edit. "Outside" the Compounds you compose the film. For example you could try out different styles of music. With smart Role-management you can turn on/off different music types, for example. important note: before export you always have to check the joints between the reels.

stay snappy:
yeah, it is the sheer amount of clips in one timeline. with feature films we always have a lot of sounddesign going on and that is hitting on performance. it's not as bad though with my Mac Studio M1 Max as with my old 5k iMac. For example: our 40 minute episodes are not a big problem any more.
With the aforementioned reels-system it's no deal at all. you could have even 3-hour-timelines. you also could temporarily put a segment into a Compound to work faster and break it apart afterwards.

stay snappier:
I'm a big fan of transcoding beforehand. So every video that goes in is ProRes and any Audio is transcoded to .wav. Old school. I heard of workflows that also work really well with (high resolution) h264-.mov-Proxies on Apple Silicon. But you have to take care that the TC runs through.

I would also recommend having a fast thunderbolt SSD-RAID or even enough internal SSD-storage (4-8 TB). Since I switched I don't have to wait for waveforms or thumbnails to built up at all, for example while zooming in or out the TL.

bug: there is also a bug, when you have long titles or effects that span over the entire timeline, such as Watermarks, TC-burnin and color adjustments. This can slow down the whole system. Try to avoid those or only apply for export. You can apply them to the "reels"-TL, without limitations.

**Comment by rodrigokirchner** [comment]
Amazing inside - we're working on a 12 episodes TV series - and we learned and verified a lot here - great to see we are on the right track :) Thank you!!

**Reply by florian-duffe** [reply]
Great to hear. If you have any questions, let us know!
